crates/bvs-pauser/src/msg.rs

```diff
@@ -4,15 +4,35 @@ use cosmwasm_schema::{cw_serde, QueryResponses};
 pub struct InstantiateMsg {
     /// Owner of this contract, who can pause and unpause
     pub owner: String,
-    /// Initial pause state
+    /// The initial paused state of satlayer contracts
     pub initial_paused: bool,
 }
 
 #[cw_serde]
 pub enum ExecuteMsg {
-    Pause {},
+    /// Callable by the owner of the pauser contract
+    Pause {
+        /// address of the contract to be paused
+        contract: String,
+        /// method of a particular contract to be paused
+        method: String,
+    },
+
+    /// Callable by the owner of the pauser contract
+    Unpause {
+        /// address of the contract to be unpaused
+        contract: String,
+        /// method of a particular contract to be unpaused
+        method: String,
+    },
+
+    /// Callable by the owner of the pauser contract
```

Comment:
```suggestion
    /// ExecuteMsg PauseGlobal Callable by the owner of the pauser contract
```

Because of the weird side-effect of how `schema.go` structure these (See schema.go and pauser.d.ts for why). Maybe write it like this?